docs/book/src/developer/providers/contracts/bootstrap-config.md

```diff
@@ -283,6 +284,12 @@ the implication of this choice which are described both in the document above an
 
 </aside>
 
+### BootstrapConfig: pausing
+
+Providers SHOULD implement the pause behaviour for every object with a reconciliation loop. This is done by checking if `spec.paused` is set on the Cluster object and by checking for the `cluster.x-k8s.io/paused` annotation on the BootstrapConfig object.
```

Comment:
Unrelated to this PR, but I just found a mistake in a previous PR

> Providers SHOULD implement the pause behaviour for every object with a reconciliation loop. This is done by checking if `spec.paused` is set on the Machine object and by checking for the `cluster.x-k8s.io/paused` annotation on the InfraMachine object.

checking if `spec.paused` is set on the Machine object

should be

checking if `spec.paused` is set on the Cluster object

Reply:
🤦 Good catch! I'll update it :)